lib/generators/pageflow/not_found/templates/404.html

```diff
@@ -3,7 +3,7 @@
 <head>
   <meta charset="utf-8"/>
   <title>Beitrag nicht gefunden (404)</title>
-  <link rel="stylesheet" type="text/css" href="error_assets/stylesheets/main.css" />
+  <link rel="stylesheet" type="text/css" href="/pageflow/stylesheets/main.css" />
```

Comment:
+1 for also using the `pageflow` namespace here. But `stylesheets/main.css` does not make is sound like these are only used for error pages. Maybe also the error pages themselves could live in `pageflow/error_pages/404.html` together with `pageflow/errors_pages/{stylesheets/main.css,images,fonts}`. That would make the scope clear and we'd still have the `public/pageflow` directory nice and clean if we ever need to add something else there.